nursery/build-docker-image.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: build Docker image
+    namespace: host-interaction/container/docker
+    author: william.ballenthin@fireeye.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Build Image on Host [T1612]
+    examples:
+    references:
+      - https://docs.docker.com/engine/api/v1.24/
+  features:
+    - or:
+      - string: /^docker(\.exe)? build/
+      - and:
+        - match: send HTTP request
+        - string: /\/v1\.[0-9]{1,2}\/build/
```

Comment:
🤮 im sorry

Reply:
its unclear to me if you can access the API without the version prefix. the docs indicate that you should send it.